lib/zenaton/traits/zenatonable.rb

```diff
@@ -22,6 +22,15 @@ def dispatch
         Engine.instance.dispatch([self])
       end
 
+      def schedule(cron)
+        if cron == '' || !cron.is_a?(String)
```

Comment:
Maybe we could use `#blank?` here instead of these two conditions. WDYT?

Reply:
`#blank?` can replace `cron == ''` but not `!cron.is_a?(String)`